src/fpga/core/rtl/chip/BSX/bsx121-124.mif

```diff
@@ -28,20 +28,20 @@ CONTENT BEGIN
 	013  :   74;
 	014  :   6F;
 	015  :   20;
-	016  :   4D;
-	017  :   69;
-	018  :   53;
-	019  :   54;
-	01A  :   65;
-	01B  :   72;
-	01C  :   21;
+	016  :   41;
```

Comment:
Add a comment (you can do that in mifs) indicating that this is the source text displayed in the games, and write out the text that this represents.